app/models/travelling.rb

```diff
@@ -1,4 +1,4 @@
 class Travelling < ApplicationRecord
   has_many :tours, dependent: :destroy
-  belongs_to :location
+  belongs_to :location, optional: true
```

Comment:
sao lại optional là true em, travelling thì phải bjk dc place mô chứ nhỉ

Reply:
Lúc đó em rails db:seed nó cứ bị lỗi location must be valid gì đó nên em mới bỏ thêm thuộc tính đó vào, bây giờ k hiểu sao em xóa optional: true đi xong chạy lại thì vẫn chạy bth :))